src/main/java/com/google/sps/server/ScheduleCalendar.java

```diff
@@ -107,12 +103,16 @@ public String getCalendarId() {
    * Helper function to add a course in the schedule to the calendar, as an event. 
    *
    * @param course the course being added 
-   * @param calendarId the id of the secondary calendar to add the course to
-   * @param startTime the start time of the course
-   * @param endTime the end time of the course
+   * @param sectionTime the timerange of the current course section
+   * @param month the current month, to be used in calculating the start and end times
+   * @param year the current year, to be used in calculating the start and end times
+   * @param nextSun the date of the next Sunday, to be used in calculating the start and end times
    */
-  private void addEvent(Course course, String startTime, String endTime) throws IOException {
+  private void addEvent(ScheduledCourse course, TimeRange sectionTime, int month, int year, int nextSun) throws IOException {
     try {  
```

Comment:
You don't necessarily need to change this now, but just a note that in general, you want to try to limit the code within the `try` block to only the code that might throw the exception.

Reply:
sorry I didn't see this before re-requesting review, but thanks for letting me know. I'll keep this in mind and change it later if I have time!